src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/CaseEventTaskHandler.java

```diff
@@ -116,6 +116,8 @@ private String getSummary(String eventId, String state) {
                     return "RPA Reason: Only one of the defendants is notified.";
                 case TAKEN_OFFLINE_BY_STAFF:
                     return "RPA Reason: Case taken offline by staff.";
+                case FULL_DEFENCE:
```

Comment:
The pattern of message is different to others, is this needed (mentions 'testing)

Reply:
need clarifications on that. I will speak to @akriti-hmcts